src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -104,11 +116,24 @@ export class MemberListComponent implements OnChanges, OnDestroy {
     private removeGroupService: RemoveGroupService,
   ) {
     this.dataFetching.pipe(
-      switchMap(params => this.getData(params.route, params.filter, params.sort).pipe(mapToFetchState({ resetter: this.refresh$ }))),
+      switchMap(params => this.getData(params).pipe(mapToFetchState({ resetter: this.refresh$ }))),
+      withLatestFrom(this.dataFetching),
     ).subscribe({
-      next: state => {
+      next: ([ state, dataFetching ]) => {
+        if (state.isError && dataFetching.fromId) {
+          this.actionFeedbackService.error($localize`Could not load more members, are you connected to the internet?`);
+          this.state = 'ready';
+          return;
+        }
         this.state = state.tag;
-        if (state.isReady) this.data = state.data;
+
+        if (state.isReady) {
+          const isLoadMore = !!dataFetching.fromId;
+          this.data = isLoadMore
+            ? { ...this.data, rowData: [ ...this.data.rowData, ...state.data.rowData ] }
+            : state.data;
+        }
+        this.loadMore = this.getLoadMore(state, dataFetching);
```

Comment:
Very good. I thought it would have been more complex and more bug-prone, it seems to work well :-).

Several thoughts though: 
- This component used the "old" way of handling data+state, in two regular variables filled in subscribe instead of full observable fetch state. So you have applied on it "old" way to handle state, so that does not apply to many components.
- Most of page handling will be the same across the components: the "load more" button state, the list appending, the "fromId" computation, the fact that error have to be handled differently, ... So probably all this "complexity" will have to be isolated in a common function or operator. 

Whatever, we can leave this PR as it is and refactor next time we apply pagination. (but maybe the 2nd case, should be something else than the progress matrix, which is a special case)